base/libgit2/blob.jl

```diff
@@ -8,10 +8,34 @@ function Base.length(blob::GitBlob)
     return ccall((:git_blob_rawsize, :libgit2), Int64, (Ptr{Void},), blob.ptr)
 end
 
+"""
+Use a heuristic to guess if a file is binary: searching for NULL bytes and
```

Comment:
On a purely pedantic note, I think it reads better if you change "searching" to "search" and "looking" to "look," since it matches "use" in the first part of the sentence.